modules/programs/git.nix

```diff
@@ -296,16 +296,75 @@ in {
           '';
         };
       };
+
+      diff-so-fancy = {
+        enable = mkEnableOption "" // {
+          description = ''
+            Enable the <command>diff-so-fancy</command> diff colorizer.
+            See <link xlink:href="https://github.com/so-fancy/diff-so-fancy" />.
+          '';
+        };
+        markEmptyLines = mkOption {
+          type = types.bool;
+          default = true;
+          example = false;
+          description = ''
+            Should the first block of an empty line be colored.
```

Comment:
```suggestion
            Should the first block of an empty line be colored?
```

Reply:
I think it feels a bit odd for the documentation to ask the user questions. I would suggest to instead use the "Whether …" kind of phrasing. E.g.

> Whether the first block of an empty line should be colored.

Reply:
> I think it feels a bit odd for the documentation to ask the user questions. I would suggest to instead use the "Whether …" kind of phrasing. E.g.
> 
> > Whether the first block of an empty line should be colored.

Yeah, I agree. I first opted to use the option descriptions from the original readme, but this seems nicer.